examples/basics/scene/mesh_normals.py

```diff
@@ -0,0 +1,45 @@
+from vispy import app, scene
```

Comment:
Could you add a license header like:

https://github.com/vispy/vispy/blob/4aad67def21b539ea1b899a43196abd188788bd1/examples/basics/scene/image.py#L1-L9

Reply:
Looks like the other new mesh examples don't have it either. If you can add it to those too that would be appreciated.

Reply:
> Could you add a license header like:
> 
> https://github.com/vispy/vispy/blob/4aad67def21b539ea1b899a43196abd188788bd1/examples/basics/scene/image.py#L1-L9

For info, what does the number after the gallery mean?

Reply:
> Looks like the other new mesh examples don't have it either. If you can add it to those too that would be appreciated.

I'll do another PR.

Reply:
The gallery number is what frame to save to an image for the gallery. These are generated with the `make gallery` command and I try to run it on larger releases to update the gallery, but since the gallery does not show all of the examples (and we haven't had a release in a while) I haven't needed to run it for a while. See here:

https://github.com/vispy/vispy/blob/4aad67def21b539ea1b899a43196abd188788bd1/make/make.py#L280-L347

Reply:
Thanks.